docs/multidatacenter.md

```diff
@@ -205,8 +214,7 @@ in your cluster as you use in the NODE_TOPOLOGY definition.
 
 ### Full list of required parameters
 
-```
-    ENDPOINT_SNITCH=GossipingPropertyFileSnitch
-    NODE_ANTI_AFFINITY=true
-    NODE_TOPOLOGY=<the cluster topology>
+```yaml
+ENDPOINT_SNITCH=GossipingPropertyFileSnitch NODE_ANTI_AFFINITY=true
+NODE_TOPOLOGY=<the cluster topology>
```

Comment:
```suggestion
```yaml
ENDPOINT_SNITCH=GossipingPropertyFileSnitch 
NODE_ANTI_AFFINITY=true
NODE_TOPOLOGY=<the cluster topology>
```

Reply:
Oops, how did that happen?

Reply:
It's the file formatter for YAML. Just noticed that this isn't actually YAML, changing this to text formatting here and in other places.